src/store/creator/selectors.ts

```diff
@@ -9,3 +9,4 @@ export const getCreatorStagedForLocation = (state: State, location: number): Cre
     return staged && staged.location === location ? staged : null;
 };
 export const getCreatorStatus = (state: State): CreatorStatus => state.creator.status;
+export const getStagedCursor = (state: State): number => state.creator.cursor;
```

Comment:
Should this be `getCreatorCursor`? It doesn't interact with the `staged` object at all.